src/lib/parsers/include/sol-json.h

```diff
@@ -433,6 +433,8 @@ char *sol_json_escape_string(const char *str, char *buf, size_t len) SOL_ATTR_NO
 
 int sol_json_double_to_str(const double value, char *buf, size_t buf_len);
 
+int sol_json_double_to_str_alloc(const double value, char **buf);
```

Comment:
-1 to this call. Even though float strings can be large, they have a relative small maximum size and by keeping those strings in the stack will avoid memory allocation, fragmentation etc.